azure-mgmt-network/HISTORY.rst

```diff
@@ -3,11 +3,36 @@
 Release History
 ===============
 
-1.8.0a1 (2017-XX-XX)
-++++++++++++++++++++
+2.0.0rc1 (2018-03-07)
++++++++++++++++++++++
+
+**Breaking changes**
+
+This version introduces a new generation of code generator that *might* introduce breaking changes. 
+
+- Model signatures are now using only keywords-arguments syntax. Every positional arguments are required to be rewritten as keywords-arguments.
+  To keep auto-completion in most cases, models are now generated for Python 2 and Python 3. Python 3 uses the "*" syntax for keyword-only arguments.
+- Enum type are now using the "str" mixin (`class AzureEnum(str, Enum)`) to improve experiences when unkown enum are met. This is not a breaking change,
+  but documentation about mixin enum should be known:
+  https://docs.python.org/3/library/enum.html#others
+  At a glance:
+
+  - "is" should not be used at all.
+  - "format" will return the string value, where "%s" string formatting will return `NameOfEnum.stringvalue`. Format syntax should be prefered.
+
+- New Long Running Operation:
+
+  - Return type changes from `msrestazure.azure_operation.AzureOperationPoller` to `msrest.polling.LROPoller`. External API is the same.
+  - Return type is now **always** a `msrest.polling.LROPoller`, whatever the optional parameters.
+  - `raw=True` changes behavior. Instead of not polling and returning the initial call as `ClientRawResponse`, now this returns a LROPoller as well and the final 
```

Comment:
- The behavior has changed when using `raw=True`. Instead of returning the initial call result as `ClientRawResponse` without polling, now this returns an LROPoller. After polling, the final resource will be returned as a `ClientRawResponse`.